src/TaCo/connectionBuilder.js

```diff
@@ -0,0 +1,43 @@
+(function dsbuilder(attr){
+    var params = {};
+
+    // Set host information in connection string
+    params["SERVER"] = attr[connectionHelper.attributeServer];
+    params["PORT"] = attr[connectionHelper.attributePort];
+
+    // Set authentication values in connection string
+    var authAttrValue = attr[connectionHelper.attributeAuthentication];
+    if (authAttrValue == "auth-none"){
```

Comment:
If we can set the auth option in `connection-dialog.tcd`, we can simplify this logic here

Reply:
done